plugins/kubernetes-backend/src/cluster-locator/GkeClusterLocator.test.ts

```diff
@@ -485,5 +486,20 @@ describe('GkeClusterLocator', () => {
         },
       ]);
     });
+    it('Check if new container.v1.ClusterManagerClient has key and values as parameter', async () => {
+      const configs: Config = new ConfigReader({
+        type: 'gke',
+        projectId: 'some-project',
+      });
+
+      const refreshIntervals: Duration | undefined = undefined;
+      const getHeaders: GkeClusterLocator = GkeClusterLocator.fromConfig(
+        configs,
+        refreshIntervals,
+      );
+
+      expect(getHeaders).toHaveProperty('client._opts.libName');
+      expect(getHeaders).toHaveProperty('client._opts.libVersion');
```

Comment:
...then your assertions can be made more robust and descriptive!
```suggestion
      GkeClusterLocator.fromConfig(configs);

      expect(container.v1.ClusterManagerClient).toHaveBeenCalledWith({
        libName: 'backstage/gke',
        libVersion: expect.any(String),
      });
```